evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -38,8 +38,7 @@ def validate(self, X, y):
         null_rows = y.isnull()
         if null_rows.any():
             messages.append(DataCheckError("{} row(s) ({}%) of target values are null".format(null_rows.sum(), null_rows.mean() * 100), self.name))
-        valid_target_types = numeric_and_boolean_dtypes + categorical_dtypes
-
+        valid_target_types = [str(dtype) for dtype in numeric_and_boolean_dtypes + categorical_dtypes]
```

Comment:
Hmm why do we need to convert the type to string now?

Reply:
Haha oops, I think this was part of my testing--don' t think we need it, will revert.